portfolio/src/main/webapp/about.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="aboutstyle.css">
```

Comment:
I didn't end up using the aboutstyle.css file and just deleted it. I guess i just forgot to delete that line in my html. 

Nonetheless, should I be making multiple css files? I would think it would increase modularity(?)